spec/lib/email/receiver_spec.rb

```diff
@@ -218,14 +218,6 @@ def create_post_reply_key(value)
     expect(IncomingEmail.last.error).to eq("RuntimeError")
   end
 
-  it "matches the correct user" do
```

Comment:
That test wasn't required (the `parse_from_field` is exercised in pretty much every other tests) and was far-reaching into internal method for no good reason.